virttest/qemu_storage.py

```diff
@@ -269,6 +269,16 @@ def _get_image_meta(image, params, root_dir):
             meta['file'].update({
                 k: v[1] for k, v in six.iteritems(mapping) if v[0]
             })
+    filters = params.get("image_filters", "").split()
```

Comment:
Confirmed with feature owners, currently, we don't support multiple filters for a single node, but I think it's OK to use a 'stack' here in case we support it in future, @YongxueHong , would you pls make some comments?

Reply:
Also, please introduce the new params in virttest/shared/cfg/base.cfg

Reply:
AFAIK, filter nodes are the first layer on the block device backend node chain. They can be used on top of the format nodes (raw, qcow2...) which are used on top of the protocol nodes (file, iscsi...). The filter nodes allow to apply different data manipulation algorithms to the data that is being read/written by the guest from/to the block device that such I/O operation is targeting.

For example, using a throttling filter over a given block device allows limiting the I/O operation that such device will be exposed to. Other example is the compress filter, which allows compressing the data that is being written onto the block device while the I/O operation is being carried out.

@zhencliu, I'd appreciate it if you could correct me if I'm wrong and add any other thing I haven't mentioned.

Reply:
@bgartzi Thanks for your detailed explanations.